keyboards/nixkeyboards/day_off/day_off.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2021 Nix Keyboards
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_all(K001, K002, K003, K004, K005, K006, K007, K008, K009, K010,\
+	       K011, K012, K013, K014, K101, K102, K103, K104, K105, K106, \
+	       K107, K108, K109, K110, K111, K112, K113, K114, K200, K201, \
+	       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, \
+	       K212, K213, K214, K300, K301, K302, K303, K304, K305, K306, \
+	       K307, K308, K309, K310, K311, K312, K313, K314, K400, K401, \
+	       K402, K403, K405, K407, K408, K410, K411, K412, K413, K414) \
+    {   {KC_NO, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014}, \
+      	{KC_NO, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114}, \
+        {K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214}, \
+	{K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314}, \
+	{K400, K401, K402, K403, KC_NO, K407, K408, KC_NO, K410, K411, K412, K413} \
+    }
```

Comment:
Have you actually tested this PR? This layout macro is not valid. There should be a 1:1 mapping between the top and bottom halves, the bottom half should match exactly the matrix rows & cols, with empty spots filled in with `KC_NO`.

Reply:
Tested just now with a keener eye. There's 3 keys not working on the bottom row. Fixing. Thank you for the keen eye.

Reply:
Pushed fix. Thanks again for the keen eye. I think I mentally fumbled when fixing whitespace.